desktop/apps/home/routes.coffee

```diff
@@ -83,6 +83,7 @@ fetchMetaphysicsData = (req)->
 
       res.locals.sd.HERO_UNITS = heroUnits
       res.locals.sd.USER_HOME_PAGE = home_page.artwork_modules
+      res.locals.sd.HIGHLIGHT_HEADER_SEARCH = req.query?.highlight_header_search
```

Comment:
Yes! This is what I was looking for. I was looking in the desktop lib and not higher up in the directory. Thanks 😄